custom-words.txt

```diff
@@ -64,4 +64,6 @@ xmldoc
 Yellowpages
 Yubico
 YubiKey
+bitwardensecret
```

Comment:
❓ Is this is marked as code with backticks where it's used will this no longer be needed?

Reply:
It's not used just as code.  We use it in a title as a descriptor of an object type.

Reply:
By "code" I meant its Markdown classification. I looked at every non-URL use of this team and think it should be surrounded by backticks.

Reply:
@differsthecat, do you agree with this?  We are using it in headings.  I don't think you can backtick in headings.

Reply:
We

Reply:
We can rewrite the heading to remove it, but it is an identifier and makes sense to me denoted as so here.

Reply:
Markdown apparently does allow backticks in headings (though it seems a bit odd), but I agree with Robyn that this is an identifier.  @withinfocus, I can absolutely make this change if you would like.  My personal opinion is that this is a valid "word", though.

Reply:
Up to your team. I guess I don't really get how this would be a traditional word.

By the way we like this list to be alphabetical too.

Reply:
I'm partial to including "bitwardensecret" in the list, as I could imagine casess where we want to reference `bitwardensecret` as code, but also "bitwardensecret" as a noun.

Reply:
Yeah, it's a bit confusing but BitwardenSecret is the name of our custom resource.

@jhbeskow  I'd place BitwardenSecret and Kubebuilder in the correct places alphabetically like Matt points out and then we can resolve this!

Reply:
> Up to your team. I guess I don't really get how this would be a traditional word.
> 
> By the way we like this list to be alphabetical too.

Yes, you are correct.  I missed that.  Fixed.